agent/bench-scripts/driver/linpack

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; tab-width: 8 -*-
+#
+# "linpack" driver script
+#
+# Responsible for transforming pbench-linpack command line inputs into the
+# the linpack invocation requirements.
+#
+# The linpack binary expects an input file with 7 lines in it:
+#
+#     Line 1: Header (disgarded)
+#     Line 2: Sub-Header (emitted as-is to stdout)
+#     Line 3: Positive integer number of tests to run
+#     line 4: Number of equations to solve (problem size)
+#     line 5: Leading dimension of array
+#     line 6: Number of trials to run
+#     line 7: Data alignment value (in Kbytes)
+#
+# For lines 4-7, if there are N tests to run, each line should have N values
+# describing the parameters for those tests. For example:
+#
+#     Line 1: Header
+#     Line 2: Sub-Header
+#     Line 3: 3
+#     line 4: 10000 20000 30000
+#     line 5: 10016 20016 30016
+#     line 6:     2     2     2
```

Comment:
Are the leading dimensions always equal to `number-of-equations-to-solve + 16`? If so, can we get rid of them?